retrofit2/src/test/java/com/linecorp/armeria/client/http/retrofit2/ArmeriaCallFactoryTest.java

```diff
@@ -309,4 +287,15 @@ public void onFailure(Call<Pojo> call, Throwable t) {
         pojoCall.cancel();
         assertThat(failCount.intValue()).isZero();
     }
+
+    @Test
+    public void respectsHttpClientUri() throws Exception {
```

Comment:
Sounds like a good suggestion. I wonder what will happen when the authority is 'group:someGroup'.